packages/scalecube-microservice/tests/implementations/Microservice.spec.ts

```diff
@@ -62,7 +73,19 @@ describe('Test the creation of Microservice', () => {
         serviceDefinition: greetingServiceDefinition,
       });
 
-      test('MethodRegistry throws an error when method reference is not a function', () => {
+      test(`
+      Scenario: Fail to register a service, reference method is not a function
+        Given   a service with definition and reference
+        And     definition and reference comply with each other
+        |service          |definition            |reference             |
+        |greetingService  |hello: RequestResponse|hello: RequestResponse|
+        |                 |greet$: RequestStream |greet$: RequestStream |
+        |                 |empty: null           |empty: null           |
+        # reference has a method that is not a function
+        When    creating a Microservice with the service
+        Then    exception will occur
```

Comment:
Add a line:
`Then  a Microservice will not be created`
`And    exception...`

Reply:
@IdoJS can you check for the relevant error that will be emitted in that case? I would like to write in the test the response as I did in other tests.